lib/hdt.js

```diff
@@ -67,6 +67,40 @@ HdtDocumentPrototype.searchTerms = function (options) {
     }, this);
   });
 };
+
+// Returns the header of the HDT document as an array of triples.
+HdtDocumentPrototype.readHeader = function () {
+  if (this.closed) return Promise.reject(new Error('The HDT document cannot be read because it is closed'));
+  return new Promise((resolve, reject) => {
+    this._readHeader((error, triples, totalCount) => {
```

Comment:
totalCount not used?

Reply:
you're right, totalcount isnt used by the js code, but it's exposed by the c++ code. Shall we just remove the totalcount param on the js-side?